python/dolfinx/geometry.py

```diff
@@ -270,3 +270,32 @@ def compute_distance_gjk(
 
     """
     return _cpp.geometry.compute_distance_gjk(p, q)
+
+def determine_point_ownership(
+    mesh: Mesh,
+    points: npt.NDArray[np.floating], 
+    cells: typing.Optional[np.ndarray] = None,
+    padding: float = 0.0
+) -> PointOwnershipData:
+    """Build PointOwnershipData for a mesh-points pair.
```

Comment:
The types of the return arguments are already documented, just say point ownership data.